Replace `JN` with `JD` text and `StratumV2` with `Stratum v2`

pavlenex opened the pull request:
It doesn't do any contextual edits. We'll have to do some work on `docs/index.md` in general as it doesn't seem accurate in general. @GitGab19 maybe this edits of /docs/index.md is something you'd like to review in the future, but for now this PR specifically addresses #175 text, image replacement will follow in another PR as I edit out the graphics.

closes #175

GitGab19 commented on src/docs/index.md:
```diff
@@ -2,34 +2,34 @@
 ### Template Providers
 
-Template Providers (TPs) are Miner-side Proxies that create custom block templates and negotiate their use with the Job Negotiator via the Job Negotiation Protocol. Template Providers are usually a Bitcoin Core full node, but can also work with a different node implementation.
+Template Providers (TPs) are Miner-side Proxies that create custom block templates and negotiate their use with the job declarator via the Job declaration Protocol. Template Providers are usually a Bitcoin Core full node, but can also work with a different node implementation.
```
Capital letter missing in "via the Job `Declaration` Protocol"

pavlenex replied:
https://github.com/stratum-mining/stratumprotocol.org/pull/182/commits/82a443ac23604eb511c76f8b8a20b2d14494bd04

GitGab19 commented on src/features.md:
```diff
@@ -98,11 +98,11 @@ Miners can choose their own work and mine their chosen transaction set in Sv2, f
 Job Selection by end-miners is an optional component of Stratum V2, separate from the main mining protocol. In fact, this is actually done by three sub-protocols:
 
-1. Job Negotiation Protocol
+1. Job declaration Protocol
```
Same here: Job `Declaration` Protocol

pavlenex replied:
https://github.com/stratum-mining/stratumprotocol.org/pull/182/commits/82a443ac23604eb511c76f8b8a20b2d14494bd04

GitGab19 commented on src/features.md:
```diff
@@ -98,11 +98,11 @@ Miners can choose their own work and mine their chosen transaction set in Sv2, f
 3. Template Distribution Protocol.
 
-Job selection is a negotiation process between miner and pool. The miner proposes a block template; the pool accepts or rejects it. Once a negotiated template is accepted, the template can be multicast to other mining devices and farms. Implementing Job Selection Protocols separately allows pools to terminate connections on seperate infrastructure from the main mining protocol, preventing impact  ...
+Job selection is a declaration process between miner and pool. The miner proposes a block template; the pool accepts or rejects it. Once a negotiated template is accepted, the template can be multicast to other mining devices and farms. Implementing Job Selection Protocols separately allows pools to terminate connections on seperate infrastructure from the main mining protocol, preventing impact  ...
```
Change in "Once a `declared` template is accepted.."

pavlenex replied:
https://github.com/stratum-mining/stratumprotocol.org/pull/182/commits/82a443ac23604eb511c76f8b8a20b2d14494bd04

GitGab19 commented on src/getting-started.md:
```diff
@@ -259,9 +259,9 @@ Eventually, the Translation Proxy log output will show sucessful share, which me
 
 
-## Config D: SV1 firmware > Translation Proxy JN (Job Negotiator) > SV2 Pool
+## Config D: SV1 firmware > Translation Proxy JN (job declarator) > SV2 Pool
```
Translation Proxy `JD (Job Declarator)`

pavlenex replied:
https://github.com/stratum-mining/stratumprotocol.org/pull/182/commits/82a443ac23604eb511c76f8b8a20b2d14494bd04

GitGab19 commented on src/getting-started.md:
```diff
@@ -259,9 +259,9 @@ Eventually, the Translation Proxy log output will show sucessful share, which me
+## Config D: SV1 firmware > Translation Proxy JN (job declarator) > SV2 Pool
 
-This configuration allows mining devices running SV1 firmware to connect to an SV2 Pool through a Translation Proxy (tProxy). In this case the tProxy is designed also to implement the **Job Negotiator (JN)** sub-protocol: allowing miners to select transactions locally and send them to the **Pool-side JN**. 
+This configuration allows mining devices running SV1 firmware to connect to an SV2 Pool through a Translation Proxy (tProxy). In this case the tProxy is designed also to implement the **job declarator (JN)** sub-protocol: allowing miners to select transactions locally and send them to the **Pool-side JN**. 
```
- to implement the `Job Declaration (JD)` sub-protocol
- them to the Pool-side `JDS (Job Declarator Server)`.

GitGab19 commented on src/getting-started.md:
```diff
@@ -259,9 +259,9 @@ Eventually, the Translation Proxy log output will show sucessful share, which me
 
-This configuration allows mining devices running SV1 firmware to connect to an SV2 Pool through a Translation Proxy (tProxy). In this case the tProxy is designed also to implement the **Job Negotiator (JN)** sub-protocol: allowing miners to select transactions locally and send them to the **Pool-side JN**. 
+This configuration allows mining devices running SV1 firmware to connect to an SV2 Pool through a Translation Proxy (tProxy). In this case the tProxy is designed also to implement the **job declarator (JN)** sub-protocol: allowing miners to select transactions locally and send them to the **Pool-side JN**. 
 In the following guide a Template Provider is installed locally on the same machine, to provide block templates to the JN.
```
JD

GitGab19 commented on src/getting-started.md:
```diff
@@ -392,7 +392,7 @@ If the pool properly starts you should see the following log lines:
 
 Once the SV2 pool is running, let's run the tProxy that will facilitate communication between the pool and a SV1 miner.
-Differently from the Config C, in this case the **tProxy** will be acting as a **Job Negotiator (JN)**, so it will select transactions locally and send them to the **Pool-side JN**
+Differently from the Config C, in this case the **tProxy** will be acting as a **job declarator (JN)**, so it will select transactions locally and send them to the **Pool-side JN**
```
```suggestion
Differently from the Config C, in this case the **tProxy** will be acting as a **Job Declarator (JD)**, so it will select transactions locally and send them to the **Pool-side JDS**
```

pavlenex replied:
https://github.com/stratum-mining/stratumprotocol.org/pull/182/commits/82a443ac23604eb511c76f8b8a20b2d14494bd04

GitGab19 reviewed (changes requested):
Just some little typos to fix

GitGab19 commented:
> It doesn't do any contextual edits. We'll have to do some work on `docs/index.md` in general as it doesn't seem accurate in general. @GitGab19 maybe this edits of /docs/index.md is something you'd like to review in the future, but for now this PR specifically addresses #175 text, image replacement will follow in another PR as I edit out the graphics.
> 
> closes #175

Agree with you. There is something to change there.
Next week I think I'll get time to have a general review and propose some changes!

pavlenex commented:
Thanks for a review @GitGab19 Addressed all your suggestions in https://github.com/stratum-mining/stratumprotocol.org/pull/182/commits/82a443ac23604eb511c76f8b8a20b2d14494bd04

There are probably a lot of things we'd have to do in GettingStarted.md as it still has a lot of mentions on JN, but let's do that in a separate PR.

pavlenex commented:
@GitGab19 I've reverted the docs/index.md page since you're working on it in #187 so I think we can merge this one.
